index.html

```diff
@@ -100,9 +98,9 @@ <h1>Professional Art Printing Data</h1>
             <h1>Professional Art Printing Data</h1>
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
             <div class="lang-btn2">
-              <div class="grid-btn1"><button type="button">Html</button></div>
-              <div class="grid-btn2"><button type="button">bootstrap</button></div>
-              <div class="grid-btn3"><button type="button">Ruby</button></div>
+              <p class="grid-btn1"><button type="button">Html</button></p>
+              <p class="grid-btn2"><button type="button">bootstrap</button></p>
+              <p class="grid-btn3"><button type="button">Ruby</button></p>
```

Comment:
Using <p> tag for organizing buttons is not a good practice. I would suggest using ul (unorder list) tags instead.

Reply:
- [ ] Good job notifying this to the student 💪 however it looks like you're missing something in this comment like `<p>` tag, try to make your comments clear and easy to understand so there will not be a miscommunication with the student. 
- [ ] Also try to explain the reason for this requirement, refer to my previous comment for an elaboration.

Reply:
Good job so for :+1: . I thing it will be better to use <div> tag instead of using <p>. It is because <p> tag is used for paragraphs or any other text.